core/src/test/scala/laserdisc/protocol/ClusterPSpec.scala

```diff
@@ -50,7 +50,7 @@ final class ClusterPSpec extends BaseSpec with ClusterP {
     )
     val rawNode = for {
       nid   <- nodeIdGen
-      h     <- Gen.oneOf(const(""), hostGen)
+      h     <- hostGen
```

Comment:
I can't find now where this is documented (assuming it is) but I suspect that I added the possibility of hosts being empty strings is because I've seen it somewhere

Reply:
I will ad that case back.

Reply:
done